Doc/library/os.rst

```diff
@@ -517,8 +521,8 @@ process and user.
    Assignments to items in ``os.environ`` are automatically translated into
    corresponding calls to :func:`putenv`; however, calls to :func:`putenv`
    don't update ``os.environ``, so it is actually preferable to assign to items
-   of ``os.environ``. This also applies to :func:`getenv` and :func:`getenvb`, which uses
-   ``os.environ`` in its implementation.
+   of ``os.environ``. This also applies to :func:`getenv` and :func:`getenvb`, which
+   respectively use ``os.environ`` and ``os.environb`` in their implementations.
```

Comment:
Alternative wording can be something like

'This also applies to getenv, which uses os.environ in its implementation (and likewise for getenvb and environb).'